uk.co.compucorp.civicrm.hrleaveandabsences/sql/auto_install.sql

```diff
@@ -391,9 +391,9 @@ SET foreign_key_checks = 0;
 INSERT INTO `civicrm_hrjobcontract` VALUES (1,202,1,0);
 INSERT INTO `civicrm_hrjobcontract_revision` VALUES (1,1,1,'2017-02-23 15:04:46','2017-02-23 15:04:47','2016-01-01','2016-01-01',NULL,NULL,1,NULL,NULL,NULL,NULL,NULL,1,1,0),(2,1,1,'2017-02-23 15:04:47','2017-02-23 15:04:48','2016-01-01','2016-01-01',NULL,NULL,1,NULL,NULL,2,NULL,NULL,1,1,0),(3,1,1,'2017-02-23 15:04:49','2017-02-23 15:04:49','2016-01-01','2016-01-01',NULL,NULL,1,NULL,3,2,NULL,NULL,1,1,0),(4,1,1,'2017-02-23 15:04:49','2017-02-23 15:04:50','2016-01-01','2016-01-01',NULL,NULL,1,NULL,3,2,4,NULL,1,1,0),(5,1,1,'2017-02-23 15:04:50','2017-02-23 15:04:50','2016-01-01','2016-01-01',NULL,NULL,1,5,3,2,4,NULL,1,1,0),(6,1,1,'2017-02-23 15:04:50','2017-02-23 15:04:50','2016-01-01','2016-01-01',NULL,NULL,1,5,3,2,4,6,1,1,0),(7,1,1,'2017-02-23 15:04:50','2017-02-23 15:04:50','2016-01-01',NULL,NULL,NULL,1,5,3,2,4,7,1,0,0);
 INSERT INTO `civicrm_hrjobcontract_leave` VALUES (1,1,20,2,1),(2,2,0,2,0),(3,3,0,2,0);
-INSERT INTO `civicrm_hrjobcontract_details` VALUES (1,NULL,NULL,NULL,NULL,'2016-01-01',NULL,NULL,0,NULL,0,NULL,NULL,1);
+INSERT INTO `civicrm_hrjobcontract_details` VALUES (1,'Developer','Developer',NULL,'Employee - Permanent','2016-01-01',NULL,NULL,0,NULL,0,NULL,NULL,1);
```

Comment:
This is for the 1.7 test and staging sites, where we need some sample data in it (mainly for demonstration purposes). When we deploy this branch to the test site, the extension will be reinstalled and this SQL script will be executed, creating the tables structure and the sample data.

This is just a temporary measure though, while the extension is still in development.  Once the extension is done, the sample data will be removed.